### PR TITLE
8/4 weeks split reintroduced

### PR DESCRIPTION
**Pull request by lehnerTgkk**
Somehow was missing in last PR

## PR Description

TBD

## Checklist
- [ x] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: docs/cloud/dev/spryker-cloud-commerce-os/preparation-for-going-live.md [hunk 1]
```diff
@@ -14,6 +14,8 @@ This document describes how to prepare a Spryker project for going live on Spryk
 
 We divided the preparation into approximate timeframes, and you can adjust them to your needs. Make sure that all the following tasks are completed one week before going live.
 
+## Eight weeks before go-live
```
Comment (dereuromark) [inline docs/cloud/dev/spryker-cloud-commerce-os/preparation-for-going-live.md:17]: The same header after each other doesnt seem to make sense

File: docs/cloud/dev/spryker-cloud-commerce-os/preparation-for-going-live.md [hunk 2]
```diff
@@ -14,6 +14,8 @@ This document describes how to prepare a Spryker project for going live on Spryk
...
+
 ## Compatibility checklist
```
Comment (michelleegan11) [inline docs/cloud/dev/spryker-cloud-commerce-os/preparation-for-going-live.md:19]:
```suggestion
## Eight weeks until five weeks before go-live: compatibility checklist
```

File: docs/cloud/dev/spryker-cloud-commerce-os/preparation-for-going-live.md (hunk 1 above)
Comment (michelleegan11) [inline docs/cloud/dev/spryker-cloud-commerce-os/preparation-for-going-live.md:17]:
```suggestion

```

File: docs/cloud/dev/spryker-cloud-commerce-os/preparation-for-going-live.md [hunk 3]
```diff
@@ -29,7 +31,6 @@ You cannot successfully deploy a project on Spryker Cloud Commerce OS unless you
 - Use the option to [test your deployments locally](/docs/scos/dev/tutorials-and-howtos/howtos/howto-do-better-deployments.html#bootstrap-with-codedeployymlcode) to understand how your application will perform and work when deployed.
 - Before deploying your payment options, test them locally. For more information, see [HowTo: Debug payment integrations locally](/docs/scos/dev/tutorials-and-howtos/howtos/howto-debug-payment-integrations-locally.html).
 
-## Until five weeks before go-live
```
Comment (michelleegan11) [inline docs/cloud/dev/spryker-cloud-commerce-os/preparation-for-going-live.md:34]:
```suggestion
Five weeks before go-live: migration plan (if applicable)
```

File: docs/cloud/dev/spryker-cloud-commerce-os/preparation-for-going-live.md (hunk 1 above)
Comment (michelleegan11) [reply docs/cloud/dev/spryker-cloud-commerce-os/preparation-for-going-live.md:17]: Please let me know if my suggestions communicate the preparation timeframes accurately

File: docs/cloud/dev/spryker-cloud-commerce-os/preparation-for-going-live.md (hunk 2 above)
Comment (lehnerTgkk) [reply docs/cloud/dev/spryker-cloud-commerce-os/preparation-for-going-live.md:19]: please just keep Eigth Weeks before go-live

File: docs/cloud/dev/spryker-cloud-commerce-os/preparation-for-going-live.md (hunk 3 above)
Comment (lehnerTgkk) [reply docs/cloud/dev/spryker-cloud-commerce-os/preparation-for-going-live.md:34]: Just keep Five Weeks Before Go-Live

File: docs/cloud/dev/spryker-cloud-commerce-os/preparation-for-going-live.md [hunk 4]
```diff
@@ -14,7 +14,9 @@ This document describes how to prepare a Spryker project for going live on Spryk
 
 We divided the preparation into approximate timeframes, and you can adjust them to your needs. Make sure that all the following tasks are completed one week before going live.
 
-## Compatibility checklist
+## Eight weeks before go-live
+
+## Eight weeks until five weeks before go-live: compatibility checklist
```
Comment (michelleegan11) [inline docs/cloud/dev/spryker-cloud-commerce-os/preparation-for-going-live.md:19]:
```suggestion
## Eight weeks before go-live
```

File: docs/cloud/dev/spryker-cloud-commerce-os/preparation-for-going-live.md [hunk 5]
```diff
@@ -29,8 +30,7 @@ You cannot successfully deploy a project on Spryker Cloud Commerce OS unless you
 - Use the option to [test your deployments locally](/docs/scos/dev/tutorials-and-howtos/howtos/howto-do-better-deployments.html#bootstrap-with-codedeployymlcode) to understand how your application will perform and work when deployed.
 - Before deploying your payment options, test them locally. For more information, see [HowTo: Debug payment integrations locally](/docs/scos/dev/tutorials-and-howtos/howtos/howto-debug-payment-integrations-locally.html).
 
-## Until five weeks before go-live
-
+Five weeks before go-live
```
Comment (michelleegan11) [inline docs/cloud/dev/spryker-cloud-commerce-os/preparation-for-going-live.md:33]:
```suggestion
## Five weeks before go-live
```

**Review by michelleegan11 — APPROVED** [review]

**Pull request merged**